pkg/utils/globalTypes/health/health.go

```diff
@@ -0,0 +1,29 @@
+package health
+
+type HealthInfo struct {
+	// Maximum HP
+	MaxHP int
+	// Border between critical and weak state
+	WeakLevel int
+	// Parameters of the updateHP function
+	Width float64
+	Tau   float64
```

Comment:
Do we want these as floats or ints?

Reply:
I would rather go for float, as they are only used for computation. That way, we avoid casting operations and longer code. But I think this is a matter of personal preferences

Reply:
Okay